### ISSUE TITLE
messed up terrain gen with TBO-classic

### ISSUE THREAD

**Issue by spyguy26** [fabric-community/spooky-time#182]
when playing with both the hallow and TBO-classic world gen seems to turn striped between hallow biomes and vanilla biomes as pictured in
![2019-11-12_18 16 22](https://user-images.githubusercontent.com/39753561/68718970-cd7da780-0578-11ea-8f9b-a522f9e39a66.png)

to reproduce have both The Hallow and tbo-classic, create a new world (as worlds created without tbo-classic are unaffected) and enter the hallow

fabric loader version: 0.6.4+build.169
fabric api version: fabric-api-0.4.1+build.245-1.14
The Hallow version: TheHallow-1.1.0-beta+1.14.4
Trinkets version: trinkets-2.1.1
Tbo-classic version: tbo-classic-1.0.0

**Comment by LemmaEOF** [comment]
I believe this would be on tbo-classic's side. It might not be properly checking which dimension it's injecting its biomes into.

**Comment by spyguy26** [comment]
all right posted an issue to TBO-Classic's issue tracker here https://github.com/valoeghese/TBOClassic/issues/1#issue-521863830

**Comment by valoeghese** [comment]
Possibly a byte overflow 🤔

**Comment by valoeghese** [comment]
Yeah almost definitely a byte overflow. (Thanks for nothing, mojang chunk nbt storage!)
If you get spam of unknown biome id in the log that’ll confirm it

**Comment by valoeghese** [comment]
Though the spam of unknown biome id that I mentioned doesn’t appear in the saved logs if I remember correctly
